editor/editor_node.cpp

```diff
@@ -1381,7 +1381,7 @@ void EditorNode::_edit_current() {
 		return;
 	}
 
-	bool capitalize = bool(EDITOR_DEF("interface/editor/capitalize_properties", true));
+	bool capitalize = bool(EDITOR_GET("interface/inspector/capitalize_properties"));
```

Comment:
Why is there both `interface/inspector/capitalize_properties` and `interface/editor/capitalize_properties`?

Reply:
```
$ rg "interface/.*/capitalize_properties"
editor/inspector_dock.cpp
556:    inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));

editor/editor_node.cpp
1384:   bool capitalize = bool(EDITOR_DEF("interface/editor/capitalize_properties", true));
4639:   EDITOR_DEF("interface/inspector/capitalize_properties", true);
```

Reply:
Sounds like it should also be `interface/inspector` in `inspector_dock.cpp`.

Reply:
> Why is there both interface/inspector/capitalize_properties and interface/editor/capitalize_properties?

`interface/editor/capitalize_properties` is wrong, if you look in code, both are used in inspector.